webdriver-spec.html

```diff
@@ -697,15 +698,26 @@
 </section> <!-- /Terminology -->
 
 <section>
-<h2>Interface</h2>
 
-<pre class=idl>partial interface Navigator {
-	[Unforgeable] readonly attribute boolean webdriver;
-};</pre>
+<h2 data-dfn-for="NavigatorAutomationInformation">Interface</h2>
 
-<p data-dfn-for="Navigator">The <dfn><code>webdriver</code></dfn> IDL attribute
- of the <code><a>Navigator</a></code> interface must return
- the value of the <dfn>webdriver-active flag</dfn>, which is initially false.
+<p>If the <dfn>webdriver-active flag</dfn> is true,
+ the user agent must support <dfn>NavigatorAutomationInformation</dfn> interface:
+
+ <pre class=idl>
+ Navigator implements NavigatorAutomationInformation;
+</pre>
+
+<pre class=idl>
+ [NoInterfaceObject, Exposed=(Window)]
+ interface NavigatorAutomationInformation {
+ [Exposed=Window] readonly attribute boolean webdriver; // constant true
```

Comment:
You don't need the additional Exposed=Window line here. I'd make the comment "always returns true" as per precedent in DOM.